app/src/services/subscribers/CensusEventSubscriberService.ts

```diff
@@ -98,16 +96,26 @@ export default class CensusEventSubscriberService implements ServiceInterface {
 
         // Set up event handlers
         this.wsClient.on('death', (event) => {
-            if (this.checkInstance(event)) {
-                const deathEvent = new DeathEvent(event, this.getInstance(event));
+            if (this.activeInstanceAuthority.instanceExists(
```

Comment:
Why are zone_id hardcoded here ? wouldnt that mean only getting from that continent ?

Reply:
You're right, merge f**ed it up